services/worker/src/worker/job_runners/config/parquet_metadata.py

```diff
@@ -31,7 +33,10 @@ def create_parquet_metadata_file_from_remote_parquet(
     parquet_file_item: SplitHubFile, fs: HTTPFileSystem, hf_token: Optional[str], parquet_metadata_directory: StrPath
 ) -> ParquetFileMetadataItem:
     try:
-        parquet_file = get_parquet_file(url=parquet_file_item["url"], fs=fs, hf_token=hf_token)
+        retry_get_parquet_file = retry(on=[aiohttp.ServerConnectionError], sleeps=HF_HUB_HTTP_ERROR_RETRY_SLEEPS)(
```

Comment:
or should it be specifically `ServerDisconnectedError`?

Reply:
if ServerConnectionError is a parent class of ServerDisconnectedError it's ok imo

Reply:
btw `retry` will do a `sleep` before each first call to `get_parquet_file`, which can make the parquet-metadata job a little bit slower (it's unlikely to be longer that the maximum allowed 40min but better keep this in mind when re-running it for datasets like fineweb)

Reply:
I changed the first value of sleeps from 1 to 0.2